orders/models.py

```diff
@@ -0,0 +1,79 @@
+from django.db import models
+from django.utils import timezone
+from daycare.models import DayCare
+from dogowner.models import DogOwner
+
+
+class StatusOptions(models.TextChoices):
+    Pending = 'P', 'Pending'
+    Approved = 'A', 'Approved'
+    Canceled = 'C', 'Canceled'
+    Finished = 'F', 'Finished'
+    OnGoing = 'O', 'On Going'
+
+
+class Order(models.Model):
+    dog_owner_id = models.ForeignKey(DogOwner, on_delete=models.CASCADE)
+    daycare_id = models.ForeignKey(DayCare, on_delete=models.CASCADE)
+    booking_date = models.DateField(auto_now=True)
+    approving_date = models.DateField(blank=True, null=True)
+    starting_date = models.DateField()
+    finishing_date = models.DateField()
+    canceling_date = models.DateField(blank=True, null=True)
+    total_price = models.IntegerField()
+    status = models.CharField(max_length=1,
+                              choices=StatusOptions.choices,
+                              default='P'
+                              )
+
+    def __str__(self):
+        return f"Dog owner ID: {self.dog_owner_id}, daycare ID: {self.daycare_id}, status: {self.status}"
+
+    @staticmethod
+    def create(dog_owner_id, daycare_id, starting_date, finishing_date, total_price):
+        new_order = Order(dog_owner_id=dog_owner_id,
+                          daycare_id=daycare_id,
+                          booking_date=timezone.now(),
+                          starting_date=starting_date,
+                          finishing_date=finishing_date,
+                          total_price=total_price,
+                          status=StatusOptions.Pending
+                          )
+
+        new_order.save()
+        return new_order
+
+    @staticmethod
+    def get_all_dog_owner_orders(user_id):
+        filter_args = {
+            'dog_owner_id': user_id,
+        }
+
+        return Order.objects.filter(**filter_args)
+
+    @staticmethod
+    def get_all_daycare_orders(user_id):
+        filter_args = {
+            'daycare_id': user_id,
+        }
+
+        return Order.objects.filter(**filter_args)
+
+    @staticmethod
+    def get_order_by_order_id(order_id):
+        return Order.objects.filter(order_id=order_id.id)
```

Comment:
Please try to keep the code more consistent, you can pass filter args directly similar to get_order_by_order_id().